closed/src/java.base/share/classes/sun/security/ec/NativeECDHKeyAgreement.java

```diff
@@ -25,27 +25,27 @@
 
 /*
  * ===========================================================================
- * (c) Copyright IBM Corp. 2022, 2023 All Rights Reserved
+ * (c) Copyright IBM Corp. 2022, 2024 All Rights Reserved
  * ===========================================================================
  */
 
 package sun.security.ec;
 
-import java.security.AlgorithmParameters;
```

Comment:
There are many differences here as compared with the result from https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/788. For example, unused imports should be removed from the head stream first so we don't need to have such differences here.

Reply:
https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/790 removes those unnecessary imports.